src/integration/pages/articles/inlineLink.js

```diff
@@ -0,0 +1,12 @@
+export default () => {
+  describe('Inline Link', () => {
+    const serviceHasInlineLink = service =>
+      service === 'news' || service === 'afaanoromoo';
```

Comment:
Do we still need this? Do all articles have an inline link or not? As an alternative to having this logic in a common / shared place, we could run the inlineLink test for the news / afaanoromoo services only

Reply:
I think you are already only running the inlineLink test for the services which need it: https://github.com/bbc/simorgh/blob/ce5c78c570b37eafbc6ca759379f5109ec70ca4f/src/integration/pages/articles/news/amp.test.js#L12, so I think it can be removed

```suggestion
```